tests/subworkflows/local/clean_reads/clean_reads.nf.test

```diff
@@ -0,0 +1,196 @@
+nextflow_workflow {
+    name "Test workflow QC_READS"
+    script "subworkflows/local/clean_reads.nf"
+    workflow "QC_READS"
+    tag "subworkflow"
+    tag "clean_reads"
+
+    test("Test clean reads run without failure illumina") {
+        tag "clean_illumina"
+
+        when {
+            workflow {
+                """
+                input[0] = Channel.of(
+                    [
+                        [id: "SAMPlE1", hybrid:
+                        false, assembly: false,
+                        downsampled: false,
+                        single_end: false,
+                        merge: false],
+                        [
+                            file("$baseDir/tests/data/reads/campy-staph1.fq.gz"),
+                            file("$baseDir/tests/data/reads/campy-staph2.fq.gz")
+                        ],
+                        "illumina"
+                    ])
+                input[1] = "illumina"
+                """
+            }
+
+            params {
+                outdir = "results"
+
+                mash_sketch = "https://github.com/phac-nml/mikrokondo/raw/dev/tests/data/databases/campy-staph-ecoli.msh"
+                mh_min_kmer = 1
+
+                dehosting_idx = "https://github.com/phac-nml/mikrokondo/raw/dev/tests/data/databases/campy.mmi"
+
+                kraken2_db = "$baseDir/tests/data/kraken2/test"
+
+
+                max_memory = "2.GB"
+                max_cpus = 1
+            }
+
+        }
+
+        then {
+            assert workflow.success
+            snapshot(workflow.out).match()
```

Comment:
Are there any other tests that could be done here? Number of reads, first read, metrics, files exist, etc.?

Same comment for the tests below.

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/125/commits/667b67d77d74bffb3d9ccedde33cd0f95ee0a188